std/digest/crc.d

```diff
@@ -381,7 +381,7 @@ struct CRC(uint N, ulong P) if (N == 32 || N == 64)
                     "1234567890123456789012345678901234567890");
     assert(digest == cast(ubyte[]) hexString!"724aa97c");
 
-    enum ubyte[4] input = hexString!"c3fcd3d7";
+    enum ubyte[4] input = cast(ubyte[4]) hexString!"c3fcd3d7";
```

Comment:
Just wondering if it is mandatory to use `enum ubyte[8] input`? `auto input` would look idiomatic.